<launch>
    <node name="gps_tracker" pkg="map_matching_localization" type="gps_tracker.py" output="screen">
        <param name="track1" type="string" value="right.csv"></param>
        <param name="track2" type="string" value="left.csv"></param>
    </node>
</launch>
